config/initializers/scimitar.rb

```diff
@@ -2,101 +2,103 @@
 #
 # For supporting information and rationale, please see README.md.
 
-# =============================================================================
-# SERVICE PROVIDER CONFIGURATION
-# =============================================================================
-#
-# This is a Ruby abstraction over a SCIM entity that declares the capabilities
-# supported by a particular implementation.
-#
-# Typically this is used to declare parts of the standard unsupported, if you
-# don't need them and don't want to provide subclass support.
-#
-Scimitar.service_provider_configuration = Scimitar::ServiceProviderConfiguration.new({
-
-  # See https://tools.ietf.org/html/rfc7643#section-8.5 for properties.
+Rails.application.config.to_prepare do
```

Comment:
So, this is great, but please can you adjust it so that the white space both in blank lines above/below and in indentation within this block, matches the Rails 7 branch here?

https://github.com/RIPAGlobal/scimitar/blob/main/config/initializers/scimitar.rb

...that way, I'll be able to update Rails 7 to get rid of the comment on the `to_prepare` line, and we'll end up with one less file that differs between the Rails 6 and 7 branches - which would be very helpful.

Thanks!

Reply:
@pond done!